README.md

```diff
@@ -1007,6 +1008,25 @@ The initialization function is run from the context of the HTML element.
 It should be noted that children nodes are removed from the DOM when widget instantiation occurs, and added as children
 to the widget instance.
 
+#### Meta
```

Comment:
Perhaps we should talk about how to create a custom meta module? Maybe show a trivial example?